src/main/kotlin/se/uulm/snowballr/backend/service/CriterionService.kt

```diff
@@ -163,8 +159,17 @@ class CriterionService(
         return criterion.toGrpcCriterion()
     }
 
-    override suspend fun createCriterion(request: CriterionOuterClass.Criterion.Create): CriterionOuterClass.Criterion =
-        repo.createCriterion(request, GrpcContext.getUserIdFromContext()).toGrpcCriterion()
+    override suspend fun createCriterion(request: CriterionOuterClass.Criterion.Create): CriterionOuterClass.Criterion {
+        val currentUser = userRepo.getUserById(GrpcContext.getUserIdFromContext())
+        if (!request.projectId.isEmpty()) {
```

Comment:
```suggestion
        if (request.projectId.isNotEmpty()) {
```

Do you use the SonarQube plugin for IntelliJ?

Reply:
No, not since I switched from the wsl Intellij to normal Intellij. Thanks for the reminder.